### ISSUE TITLE
2025-03-07, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#104]
# Today's Arxiv Papers

(1) [L$^2$M: Mutual Information Scaling Law for Long-Context Language Modeling](http://arxiv.org/abs/2503.04725v1), 2025-03-06

(2) [LLMVoX: Autoregressive Streaming Text-to-Speech Model for Any LLM](http://arxiv.org/abs/2503.04724v1), 2025-03-06

(3) [Shifting Long-Context LLMs Research from Input to Output](http://arxiv.org/abs/2503.04723v1), 2025-03-06

(4) [Enough Coin Flips Can Make LLMs Act Bayesian](http://arxiv.org/abs/2503.04722v1), 2025-03-06

(5) [Predictable Scale: Part I -- Optimal Hyperparameter Scaling Law in Large Language Model Pretraining](http://arxiv.org/abs/2503.04715v1), 2025-03-06

(6) [Universality of Layer-Level Entropy-Weighted Quantization Beyond Model Architecture and Size](http://arxiv.org/abs/2503.04704v1), 2025-03-06

(7) [UIPE: Enhancing LLM Unlearning by Removing Knowledge Related to Forgetting Targets](http://arxiv.org/abs/2503.04693v1), 2025-03-06

(8) [Quantifying the Reasoning Abilities of LLMs on Real-world Clinical Cases](http://arxiv.org/abs/2503.04691v1), 2025-03-06

(9) [DIMSUM: Discourse in Mathematical Reasoning as a Supervision Module](http://arxiv.org/abs/2503.04685v1), 2025-03-06

(10) [LLM-guided Plan and Retrieval: A Strategic Alignment for Interpretable User Satisfaction Estimation in Dialogue](http://arxiv.org/abs/2503.04675v1), 2025-03-06

(11) [Implicit Cross-Lingual Rewarding for Efficient Multilingual Preference Alignment](http://arxiv.org/abs/2503.04647v1), 2025-03-06

(12) [IFIR: A Comprehensive Benchmark for Evaluating Instruction-Following in Expert-Domain Information Retrieval](http://arxiv.org/abs/2503.04644v1), 2025-03-06

(13) [Mark Your LLM: Detecting the Misuse of Open-Source Large Language Models via Watermarking](http://arxiv.org/abs/2503.04636v1), 2025-03-06

(14) [SurveyForge: On the Outline Heuristics, Memory-Driven Generation, and Multi-dimensional Evaluation for Automated Survey Writing](http://arxiv.org/abs/2503.04629v1), 2025-03-06

(15) [START: Self-taught Reasoner with Tools](http://arxiv.org/abs/2503.04625v1), 2025-03-06

(16) [SynGraph: A Dynamic Graph-LLM Synthesis Framework for Sparse Streaming User Sentiment Modeling](http://arxiv.org/abs/2503.04619v1), 2025-03-06

(17) [Better Process Supervision with Bi-directional Rewarding Signals](http://arxiv.org/abs/2503.04618v1), 2025-03-06

(18) [HalluCounter: Reference-free LLM Hallucination Detection in the Wild!](http://arxiv.org/abs/2503.04615v1), 2025-03-06

(19) [Towards Data-Efficient Language Models: A Child-Inspired Approach to Language Learning](http://arxiv.org/abs/2503.04611v1), 2025-03-06

(20) [HybridNorm: Towards Stable and Efficient Transformer Training via Hybrid Normalization](http://arxiv.org/abs/2503.04598v1), 2025-03-06

(21) [The Next Frontier of LLM Applications: Open Ecosystems and Hardware Synergy](http://arxiv.org/abs/2503.04596v1), 2025-03-06

(22) [Learning Generalizable Language-Conditioned Cloth Manipulation from Long Demonstrations](http://arxiv.org/abs/2503.04557v1), 2025-03-06

(23) [Compositional Translation: A Novel LLM-based Approach for Low-resource Machine Translation](http://arxiv.org/abs/2503.04554v1), 2025-03-06

(24) [Benchmarking Reasoning Robustness in Large Language Models](http://arxiv.org/abs/2503.04550v1), 2025-03-06

(25) [An Empirical Study on Eliciting and Improving R1-like Reasoning Models](http://arxiv.org/abs/2503.04548v1), 2025-03-06

(26) [Keeping Yourself is Important in Downstream Tuning Multimodal Large Language Model](http://arxiv.org/abs/2503.04543v1), 2025-03-06

(27) [SOLAR: Scalable Optimization of Large-scale Architecture for Reasoning](http://arxiv.org/abs/2503.04530v1), 2025-03-06

(28) [Multi-modal Summarization in Model-Based Engineering: Automotive Software Development Case Study](http://arxiv.org/abs/2503.04506v1), 2025-03-06

(29) [Large Language Models in Bioinformatics: A Survey](http://arxiv.org/abs/2503.04490v1), 2025-03-06

(30) [ToolFuzz -- Automated Agent Tool Testing](http://arxiv.org/abs/2503.04479v1), 2025-03-06

(31) [Know Thy Judge: On the Robustness Meta-Evaluation of LLM Safety Judges](http://arxiv.org/abs/2503.04474v1), 2025-03-06

(32) [Guiding LLMs to Generate High-Fidelity and High-Quality Counterfactual Explanations for Text Classification](http://arxiv.org/abs/2503.04463v1), 2025-03-06

(33) [TPC: Cross-Temporal Prediction Connection for Vision-Language Model Hallucination Reduction](http://arxiv.org/abs/2503.04457v1), 2025-03-06

(34) [ToFu: Visual Tokens Reduction via Fusion for Multi-modal, Multi-patch, Multi-image Task](http://arxiv.org/abs/2503.04444v1), 2025-03-06

(35) [Activation Space Interventions Can Be Transferred Between Large Language Models](http://arxiv.org/abs/2503.04429v1), 2025-03-06

(36) [AOLO: Analysis and Optimization For Low-Carbon Oriented Wireless Large Language Model Services](http://arxiv.org/abs/2503.04418v1), 2025-03-06

(37) [Can Large Language Models Predict Antimicrobial Resistance Gene?](http://arxiv.org/abs/2503.04413v1), 2025-03-06

(38) [Wider or Deeper? Scaling LLM Inference-Time Compute with Adaptive Branching Tree Search](http://arxiv.org/abs/2503.04412v1), 2025-03-06

(39) [Speculative MoE: Communication Efficient Parallel MoE Inference with Speculative Token and Expert Pre-scheduling](http://arxiv.org/abs/2503.04398v1), 2025-03-06

(40) [TableLoRA: Low-rank Adaptation on Table Structure Understanding for Large Language Models](http://arxiv.org/abs/2503.04396v1), 2025-03-06

(41) [Shaping Shared Languages: Human and Large Language Models' Inductive Biases in Emergent Communication](http://arxiv.org/abs/2503.04395v1), 2025-03-06

(42) [AgentSafe: Safeguarding Large Language Model-based Multi-agent Systems via Hierarchical Data Management](http://arxiv.org/abs/2503.04392v1), 2025-03-06

(43) [More Documents, Same Length: Isolating the Challenge of Multiple Documents in RAG](http://arxiv.org/abs/2503.04388v1), 2025-03-06

(44) [TRACT: Regression-Aware Fine-tuning Meets Chain-of-Thought Reasoning for LLM-as-a-Judge](http://arxiv.org/abs/2503.04381v1), 2025-03-06

(45) [How can representation dimension dominate structurally pruned LLMs?](http://arxiv.org/abs/2503.04377v1), 2025-03-06

(46) [Lost in Literalism: How Supervised Training Shapes Translationese in LLMs](http://arxiv.org/abs/2503.04369v1), 2025-03-06

(47) [Exploring the Multilingual NLG Evaluation Abilities of LLM-Based Evaluators](http://arxiv.org/abs/2503.04360v1), 2025-03-06

(48) [Layer-Specific Scaling of Positional Encodings for Superior Long-Context Modeling](http://arxiv.org/abs/2503.04355v1), 2025-03-06

(49) [Large Language Models for Zero-shot Inference of Causal Structures in Biology](http://arxiv.org/abs/2503.04347v1), 2025-03-06

(50) [In-depth Analysis of Graph-based RAG in a Unified Framework](http://arxiv.org/abs/2503.04338v1), 2025-03-06

(51) [The Challenge of Identifying the Origin of Black-Box Large Language Models](http://arxiv.org/abs/2503.04332v1), 2025-03-06

(52) [Solving Word-Sense Disambiguation and Word-Sense Induction with Dictionary Examples](http://arxiv.org/abs/2503.04328v1), 2025-03-06

(53) [Malware Detection at the Edge with Lightweight LLMs: A Performance Evaluation](http://arxiv.org/abs/2503.04302v1), 2025-03-06

(54) [Mapping AI Benchmark Data to Quantitative Risk Estimates Through Expert Elicitation](http://arxiv.org/abs/2503.04299v1), 2025-03-06

(55) [MathMistake Checker: A Comprehensive Demonstration for Step-by-Step Math Problem Mistake Finding by Prompt-Guided LLMs](http://arxiv.org/abs/2503.04291v1), 2025-03-06

(56) [How Do Hackathons Foster Creativity? Towards AI Collaborative Evaluation of Creativity at Scale](http://arxiv.org/abs/2503.04290v1), 2025-03-06

(57) [Towards Autonomous Reinforcement Learning for Real-World Robotic Manipulation with Large Language Models](http://arxiv.org/abs/2503.04280v1), 2025-03-06

(58) [On Fact and Frequency: LLM Responses to Misinformation Expressed with Uncertainty](http://arxiv.org/abs/2503.04271v1), 2025-03-06

(59) [VirtualXAI: A User-Centric Framework for Explainability Assessment Leveraging GPT-Generated Personas](http://arxiv.org/abs/2503.04261v1), 2025-03-06

(60) [ADOR: A Design Exploration Framework for LLM Serving with Enhanced Latency and Throughput](http://arxiv.org/abs/2503.04253v1), 2025-03-06

(61) [An Egocentric Vision-Language Model based Portable Real-time Smart Assistant](http://arxiv.org/abs/2503.04250v1), 2025-03-06

(62) [How to Mitigate Overfitting in Weak-to-strong Generalization?](http://arxiv.org/abs/2503.04249v1), 2025-03-06

(63) [ThrowBench: Benchmarking LLMs by Predicting Runtime Exceptions](http://arxiv.org/abs/2503.04241v1), 2025-03-06

(64) [DiffPO: Diffusion-styled Preference Optimization for Efficient Inference-Time Alignment of Large Language Models](http://arxiv.org/abs/2503.04240v1), 2025-03-06

(65) [SemaSK: Answering Semantics-aware Spatial Keyword Queries with Large Language Models](http://arxiv.org/abs/2503.04234v1), 2025-03-06

(66) [FuseChat-3.0: Preference Optimization Meets Heterogeneous Model Fusion](http://arxiv.org/abs/2503.04222v1), 2025-03-06

(67) [Knowledge-Decoupled Synergetic Learning: An MLLM based Collaborative Approach to Few-shot Multimodal Dialogue Intention Recognition](http://arxiv.org/abs/2503.04201v1), 2025-03-06

(68) [MASTER: Multimodal Segmentation with Text Prompts](http://arxiv.org/abs/2503.04199v1), 2025-03-06

(69) [Measuring temporal effects of agent knowledge by date-controlled tool use](http://arxiv.org/abs/2503.04188v1), 2025-03-06

(70) [TIMER: Temporal Instruction Modeling and Evaluation for Longitudinal Clinical Records](http://arxiv.org/abs/2503.04176v1), 2025-03-06

(71) [Semantic Retrieval Augmented Contrastive Learning for Sequential Recommendation](http://arxiv.org/abs/2503.04162v1), 2025-03-06

(72) [KidneyTalk-open: No-code Deployment of a Private Large Language Model with Medical Documentation-Enhanced Knowledge Database for Kidney Disease](http://arxiv.org/abs/2503.04153v1), 2025-03-06

(73) [Ticktack : Long Span Temporal Alignment of Large Language Models Leveraging Sexagenary Cycle Time Expression](http://arxiv.org/abs/2503.04150v1), 2025-03-06

(74) [Dynamic Benchmarking of Reasoning Capabilities in Code Large Language Models Under Data Contamination](http://arxiv.org/abs/2503.04149v1), 2025-03-06

(75) [HEISIR: Hierarchical Expansion of Inverted Semantic Indexing for Training-free Retrieval of Conversational Data using LLMs](http://arxiv.org/abs/2503.04141v1), 2025-03-06

(76) [Biological Sequence with Language Model Prompting: A Survey](http://arxiv.org/abs/2503.04135v1), 2025-03-06

(77) [Token-Efficient Long Video Understanding for Multimodal LLMs](http://arxiv.org/abs/2503.04130v1), 2025-03-06

(78) [Uncovering Gaps in How Humans and LLMs Interpret Subjective Language](http://arxiv.org/abs/2503.04113v1), 2025-03-06

(79) [InterChat: Enhancing Generative Visual Analytics using Multimodal Interactions](http://arxiv.org/abs/2503.04110v1), 2025-03-06

(80) [LLMs Can Generate a Better Answer by Aggregating Their Own Responses](http://arxiv.org/abs/2503.04104v1), 2025-03-06

(81) [Disparities in LLM Reasoning Accuracy and Explanations: A Case Study on African American English](http://arxiv.org/abs/2503.04099v1), 2025-03-06

(82) [Chart-HQA: A Benchmark for Hypothetical Question Answering in Charts](http://arxiv.org/abs/2503.04095v1), 2025-03-06

(83) [PokéChamp: an Expert-level Minimax Language Agent](http://arxiv.org/abs/2503.04094v1), 2025-03-06

(84) [Beyond Memorization: Evaluating the True Type Inference Capabilities of LLMs for Java Code Snippets](http://arxiv.org/abs/2503.04076v1), 2025-03-06

(85) [PP-DocBee: Improving Multimodal Document Understanding Through a Bag of Tricks](http://arxiv.org/abs/2503.04065v1), 2025-03-06

(86) [Uncovering inequalities in new knowledge learning by large language models across different languages](http://arxiv.org/abs/2503.04064v1), 2025-03-06

(87) [EVE: Towards End-to-End Video Subtitle Extraction with Vision-Language Models](http://arxiv.org/abs/2503.04058v1), 2025-03-06

(88) [Insights from Rights and Wrongs: A Large Language Model for Solving Assertion Failures in RTL Design](http://arxiv.org/abs/2503.04057v1), 2025-03-06

(89) [Robust Data Watermarking in Language Models by Injecting Fictitious Knowledge](http://arxiv.org/abs/2503.04036v1), 2025-03-06

(90) [Benchmarking Large Language Models on Multiple Tasks in Bioinformatics NLP with Prompting](http://arxiv.org/abs/2503.04013v1), 2025-03-06

(91) [DSV-LFS: Unifying LLM-Driven Semantic Cues with Visual Features for Robust Few-Shot Segmentation](http://arxiv.org/abs/2503.04006v1), 2025-03-06

(92) [RetinalGPT: A Retinal Clinical Preference Conversational Assistant Powered by Large Vision-Language Models](http://arxiv.org/abs/2503.03987v1), 2025-03-06

(93) [ReasonGraph: Visualisation of Reasoning Paths](http://arxiv.org/abs/2503.03979v1), 2025-03-06

(94) [Trim My View: An LLM-Based Code Query System for Module Retrieval in Robotic Firmware](http://arxiv.org/abs/2503.03969v1), 2025-03-05

(95) [Model Behavior Specification by Leveraging LLM Self-Playing and Self-Improving](http://arxiv.org/abs/2503.03967v1), 2025-03-05

(96) [Performance Comparison of Large Language Models on Advanced Calculus Problems](http://arxiv.org/abs/2503.03960v1), 2025-03-05

(97) [Tec-Habilidad: Skill Classification for Bridging Education and Employment](http://arxiv.org/abs/2503.03932v1), 2025-03-05

(98) [Safe LLM-Controlled Robots with Formal Guarantees via Reachability Analysis](http://arxiv.org/abs/2503.03911v1), 2025-03-05

(99) [Pretrained LLMs as Real-Time Controllers for Robot Operated Serial Production Line](http://arxiv.org/abs/2503.03889v1), 2025-03-05

(100) [LEWIS (LayEr WIse Sparsity) -- A Training Free Guided Model Merging Approach](http://arxiv.org/abs/2503.03874v1), 2025-03-05
